manual/statements/built/README.md

```diff
@@ -1,5 +1,269 @@
 ## Built statements
 
-*Coming soon... In the meantime, see the javadoc for [QueryBuilder].*
+Built statements are generated via [QueryBuilder]'s Fluent API.  Use of Fluent API allows
+easier build of complex queries, as opposed to use of hardcoded query strings.
 
-[QueryBuilder]: http://docs.datastax.com/en/drivers/java/3.5/com/datastax/driver/core/querybuilder/QueryBuilder.html
\ No newline at end of file
+Note: The provided builders perform very little validation of the built query. There is
+no guarantee that a built query is valid, and it is definitively possible to create
+invalid queries.
+
+Queries built with `QueryBuilder` are executed the same way as other queries--via
+`execute` or `executeAsync`.  When query is built with inlined values, then it doesn't
+differ much from a statement specified as a string.  But it's also possible to build
+the query with bind markers inside it, and then convert it into a [prepared statement](../prepared/).
+
+### Basics
+
+Generation of `BuiltStatement` is easy--start by calling of one of the
+[QueryBuilder]'s methods of that represent the CQL's "verb": `select`, `update`, `delete`,
+`insertInto`, or `truncate`, provide required parameters, and then call "verb"-specific
+functions to form a complete CQL statement (like, `where`, `from`, etc.).  The statement's
+target table can be specified as a simple table name (if a default keyspace has been set
+when creating the `Session` object), as a combination of keyspace name and table name, or as
+a [TableMetadata] object.
+
+Note: The `QueryBuilder` doesn't provide support for the full set of CQL.  For
+DDL operations (`CREATE TABLE`, etc.) you can use the [SchemaBuilder].  To perform other
+operations you still need to use [simple statements](../simple/).
```

Comment:
Any example of queries that are neither covered by `QueryBuilder` nor `SchemaBuilder`? We try to stay pretty up-to-date...

Reply:
for example, there is no support for `ALTER TYPE`,`CREATE MATERIALIZED VIEW`, etc.